lib/presentation/widgets/details_content.dart

```diff
@@ -0,0 +1,150 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/painting.dart';
+import 'package:marvelapp_flutter/presentation/widgets/series_element.dart';
+import '../../data/model/response_models/character.dart';
+import '../../data/model/response_models/series.dart';
+
+class DetailsContent extends StatelessWidget {
+  final Character? character;
+  final List<Series>? series;
+
+  const DetailsContent(
+      {Key? key, required this.character, required this.series})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    if (character == null) {
+      return const Text(
+        "Information not available",
+        style: TextStyle(
+          fontSize: 16,
+          fontWeight: FontWeight.bold,
+        ),
+      );
+    } else {
+      return Padding(
+        padding: const EdgeInsets.all(6.0),
+        child: Column(
+          verticalDirection: VerticalDirection.down,
+          children: <Widget>[
+            FadeInImage.assetNetwork(
+              placeholder: "assets/graphics/placeholder-150x150.png",
+              image: _getThumbnailUrl(),
+              height: 300,
+              width: 300,
+              fit: BoxFit.fitHeight,
+            ),
+            Padding(
+              padding: const EdgeInsets.all(8.0),
+              child: Row(
+                mainAxisAlignment: MainAxisAlignment.start,
+                children: [
+                  Text(
+                    character?.name ?? "missing name",
+                    style: const TextStyle(
+                      fontWeight: FontWeight.bold,
+                      fontSize: 16.0,
+                    ),
+                  ),
+                ],
+              ),
+            ),
+            _getDescriptionBlock(),
+            _getSeriesBlock(),
+          ],
+        ),
+      );
+    }
+  }
+
+  String _getThumbnailUrl() {
+    String? path = character?.thumbnailPath;
+    String? extension = character?.thumbnailExtension;
+    String format = "standard_xlarge";
+    return "$path/$format.$extension";
+  }
+
+  Widget _getDescriptionBlock() {
+    bool visible = false;
+    print(character?.description);
+    if (character?.description != null && character?.description != "") {
```

Comment:
Visibility have to be handled on higher level, please read my comment above.

Reply:
such chek is right? if (series != null && series!.isNotEmpty)   if series equal null series!.isNotEmpty is not cheking